### PR TITLE
Use fixed indentation when generating fields.asciidoc

### PR DESCRIPTION
**Pull request by monicasarbu**
When generating the fields docs use "===" for the fields group section and "====" for each field. Basically I rolled-back to the previous version of the script.

### CODE REVIEW

**Comment by monicasarbu** [comment]
Here is the screenshot with the resulted asciidoc.

screen shot 2016-02-01 at 11 33 44 pm

File: libbeat/scripts/generate_fields_docs.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+import sys
 
-def document_fields(output, section, SECTIONS, level=3):
+
+def document_fields(output, section, SECTIONS):
 
     if "anchor" in section:
         output.write("[[exported-fields-{}]]\n".format(section["anchor"]))
-    output.write("{} {} Fields\n\n".format(level*'=', section["name"]))
+    output.write("=== {} Fields\n\n".format(section["name"]))
```
Comment (ruflin) [inline libbeat/scripts/generate_fields_docs.py]: Be aware, that the beats had different implementations here:

with === and levels
* winlogbeat: https://github.com/elastic/beats/pull/890/files#diff-7ad2172f5c2f9e52b7c9c5bc6802e56aL22
* filebeat: https://github.com/elastic/beats/pull/890/files#diff-fb727cf14fe8bf8169f4ab4d7cce8cc6R7

with {} and without levels:
* packetbeat: https://github.com/elastic/beats/pull/890/files#diff-78627e2210ed02aff6673f49d77ea60dL28

with === but without levels:
* topbeat: https://github.com/elastic/beats/pull/890/files#diff-4089deb253efa80c804e59ed8af3a1a4L23

I tried to unify these behaviours with the levels.

**Comment by ruflin** [comment]
@monicasarbu Did you run `make update` to check what the affects are on winlogbeat and filebeat?

**Comment by monicasarbu** [comment]
@ruflin There are no changes for winlogbeat & filebeat. I checked the generated docs and it looks ok.

**Comment by ruflin** [comment]
LGTM. So the hierarchy in the docs will not represent the hierarchy in the yaml file.

**Comment by monicasarbu** [comment]
Yes, exactly.

**Pull request merged**